wandb/sdk/internal/internal_api.py

```diff
@@ -96,6 +97,45 @@ class DefaultSettings(TypedDict):
 #     def __getitem__(self, name: str) -> Any: ...
 
 
+def _guess_response_content(response: Any) -> bytes:
```

Comment:
wow... that sucks that we have to do this :(
but based on your description there is nothing better here

Reply:
Yeah :( Bummer.